VERSIONS

```diff
@@ -9,6 +9,7 @@ OPENSSL_LINUX   "1.1.1q"
 OPENSSL_MACOS   "1.1.1q"
 OPENSSL_WINDOWS "1.1.1q"
 USE_RCLONE "true"
+RCLONE_VERSION "1.59.0"
```

Comment:
Was rclone not listed here before?

Reply:
It was always the same binary of 1.51.0 version so there was no need to implicitly specify it per branch.